flocker/node/testtools.py

```diff
@@ -18,7 +18,7 @@
 
 from characteristic import attributes
 
-from twisted.trial.unittest import TestCase, SkipTest
+from twisted.trial.unittest import SkipTest
```

Comment:
Any or either would work. I vaguely prefer `self.skipTest`. In this case, it has to be the exception, since it's raised in a decorator without access to the `TestCase`. We could refactor it to use `@skipUnless` but later.